Scripts/signFunction.js

```diff
@@ -0,0 +1,48 @@
+import {formatCheck} from './accountData.js'
+
+const errorType = {
+   email : '이메일을 입력해 주세요.',
+   password : '비밀번호를 입력해 주세요.',
+   mismatchEmail : '이메일을 확인해 주세요.',
+   notIdentified : '식별되지 않은 로직 오류',
+   mismatchPassword : '비밀번호를 확인해 주세요.',
+   registeredEmail : '이미 사용중인 이메일입니다.',   
+   emailNotValid : '올바른 이메일 주소가 아닙니다.',
+   mismatchPasswordCheck : '비밀번호가 일치하지 않아요.',
+   passwordNotValid : '비밀번호는 영문, 숫자 조합 8자 이상 입력해 주세요.'
+};
```

Comment:
자바스크립트에서 상수는 일반적으로 대문자로 명명합니다.

```suggestion
const INPUT_ERROR = {
   EMAIL : '이메일을 입력해 주세요.',
   PASSWORD : '비밀번호를 입력해 주세요.',
   MISMATCHED_EMAIL : '이메일을 확인해 주세요.',
   NOT_IDENTIFIED : '식별되지 않은 로직 오류',
   ...
};
```